spec/models/event_spec.rb

```diff
@@ -24,7 +24,7 @@
                             .with_options(null: false) }
 
     it { is_expected.to have_db_column(:hash_tag).of_type(:string)
-                            .with_options(limit: 30, null: false) }
+                            .with_options(limit: 75, null: false) }
```

Comment:
Expression at 27, 67 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 24) spaces for indenting an expression spanning multiple lines.